### PR TITLE
feat(virtctl): Add documentation for the virtctl create vm command

### PR DESCRIPTION
**Pull request by 0xFelix**
This adds documentation for the virtctl create vm command.

### CODE REVIEW

File: docs/virtual_machines/creating_vms.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+# Creating VirtualMachines
+
+The virtctl sub command `create vm` allows easy creation of virtual machine
+manifests from the command line. It leverages instance types and preferences
```
Comment (lyarwood) [inline docs/virtual_machines/creating_vms.md:4]: nit - link?

File: docs/virtual_machines/creating_vms.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+# Using a cluster scoped instancetype and a namespaced preference
+virtctl create vm \
+  --instancetype virtualmachineclusterinstancetype/my-instancetype \
+  --preference virtualmachinepreference/my-preference
+```
+
+If a prefix was not supplied the cluster scoped resources will be used by
+default.
+
+To infer instance types and/or preferences from the volume used to boot the
```
Comment (lyarwood) [inline docs/virtual_machines/creating_vms.md:41]: nit - if it lands before this link to https://github.com/kubevirt/user-guide/pull/629/commits/2ca15b6a3bfc4dfbde76b01048c76ca9b4641ceb

File: docs/virtual_machines/creating_vms.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+virtctl create vm --infer-instancetype --infer-preference
+```
+
+## Boot order of added volumes
+
+Please note that volumes of different kinds currently have the following fixed
+boot order regardless of the order their flags were specified on the
+command line:
+
+```
```
Comment (lyarwood) [inline docs/virtual_machines/creating_vms.md:54]: Would a numbered list be better here?

**Review by lyarwood — CHANGES_REQUESTED** [review]

File: docs/virtual_machines/creating_vms.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+The virtctl sub command `create vm` allows easy creation of virtual machine
+manifests from the command line. It
+leverages [instance types and preferences](./instancetypes.md)
+and provides several flags to control details of the created virtual machine.
+
+For example there are flags to specify the name or run strategy of a
+virtual machine or flags to add volumes to a virtual machine. Instance types
+and preferences can either be specified directly or it is possible to let
+KubeVirt derive those from the volume used to boot the virtual machine.
```
Comment (lyarwood) [inline docs/virtual_machines/creating_vms.md:11]: While derive is valid I'd use the same term as the flag here, infer.

File: docs/virtual_machines/creating_vms.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+virtctl create vm --infer-instancetype --infer-preference
+```
+
+## Boot order of added volumes
+
+Please note that volumes of different kinds currently have the following fixed
+boot order regardless of the order their flags were specified on the
+command line:
+
+1. Containerdisk
```
Comment (lyarwood) [inline docs/virtual_machines/creating_vms.md:56]: Mixed case, can you capitalise?

**Review by lyarwood — CHANGES_REQUESTED** [review]
Looking good, could you also add an example piping the definition into `kubectl apply -f` just to make it obvious to users?

**Comment by 0xFelix** [comment]
@lyarwood Addressed your comments.

**Review by jean-edouard — COMMENTED** [review]
/approve

**Pull request merged**